static/src/javascripts/projects/common/modules/analytics/coreVitals.ts

```diff
@@ -30,8 +30,16 @@ const jsonData: CoreWebVitalsPayload = {
 };
 
 export const coreVitals = (): void => {
-	const inSample = Math.floor(Math.random() * 100);
-	if (inSample != 1) {
+	// If the current user is part of a server-side AB test
+	// then we always want to sample their core web vitals data.
+	const userInTest =
+		window.guardian.config.tests &&
+		Object.values(window.guardian.config.tests).includes('variant');
+
+	// Otherwise, only send core web vitals data for 1% of users.
+	const inSample = Math.floor(Math.random() * 100) == 1;
```

Comment:
To me this definition of `inSample` makes the variable's name more accurate.

Reply:
I would suggest the following, which allows to easily edit the right-hand-side to any other percentage value:

```suggestion
	const inSample = Math.random() < 1 / 100;
```